recipe/meta.yaml

```diff
@@ -1,22 +1,20 @@
-{% set name = "gts" %}
-{% set version = "0.7.6" %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: gts
+  version: 0.7.6
 
 source:
-  - url: http://downloads.sourceforge.net/gts/gts-{{ version }}.tar.gz
+  - url: http://iweb.dl.sourceforge.net/project/gts/gts/0.7.6/gts-0.7.6.tar.gz
```

Comment:
Could we make this 

    http://iweb.dl.sourceforge.net/project/gts/gts/{{ version }}/gts-{{ version }}.tar.gz

?